### ISSUE TITLE
Dimension in the softmax

### ISSUE THREAD

**Issue by carlogarro** [qubvel/segmentation_models.pytorch#143]
Hello, it seems that now in when calculating the softmax, the dimension must be selected. So this should be fixed.

`UserWarning: Implicit dimension choice for softmax has been deprecated. Change the call to include dim=X as an argument.`

Thanks.

**Comment by qubvel** [comment]
Thanks, will fix it!

**Comment by gireeshkbogu** [comment]
Seems I'm having the same issue.

**Comment by mccrinbc** [comment]
Is there a workaround for this in the meantime? Is there a way to edit the classification head to add the dimension?

**Comment by seba-eng** [comment]
any news on that? It's probably not a big deal, at least I don't experience any problems. But getting the warning every time is a bit annoying.

**Comment by qubvel** [comment]
`softmax2d` have been added, it uses `dim=1`
